homeconnect_flutter/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: homeconnect_flutter
-description: A starting point for Dart libraries or applications.
-version: 0.0.1
+description: Flutter classes to be used with the home connect SDK
```

A little longer description so it passes the requirements
```suggestion
description: Flutter utility classes, meant to be used with the home connect SDK
```